src/mudlet-lua/lua/geyser/GeyserLabel.lua

```diff
@@ -95,6 +99,19 @@ function Geyser.Label:processFormatString(format)
   end
 end
 
+--- Sets the font face for the label, use empty string to clear the font and use css/default. Returns true if the font changed, nil+error if not.
+-- @param font font face to use
+function Geyser.Label:setFont(font)
+  local af = getAvailableFonts()
+  if not (af[font] or font == "") then
+    local err = "attempt to call setFont with font '" .. font .. "' which is not available, see getAvailableFonts() for valid options\n"
+    err = err .. "In the meantime, Qt will use some font which isn't the one you asked for but we hope is close enough"
```

Comment:
```suggestion
    err = err .. "In the meantime, we'll will use some font which isn't the one you asked for but we hope is close enough"
```
Not everyone would know what Qt is or need to, less confusion on the implementation detail